README.md

```diff
@@ -65,6 +65,7 @@ aa('setUserToken', 'USER_ID');
 | **`apiKey`**      | `string`       | None (required)          | The search API key of your Algolia application |
 | `userHasOptedOut` | `boolean`      | `false`                  | Whether to exclude users from analytics        |
 | `region`          | `'de' \| 'us'` | Automatic                | The DNS server to target                       |
+| `useCookie`       | `boolean`      | `true`                   | Whether to use cookie in browser environment. The anonymous user token will not be set if `false`. |
```

Comment:
after this, I think we should release a version with useCookie as true

Reply:
You mean `false` as a breaking change? It's true by default.

Reply:
Yes, sorry I meant `false` in the future

Reply:
yeah it should be `false` in the next major version.